sonar-db/src/test/java/org/sonar/db/charset/SqlExecutorTest.java

```diff
@@ -44,21 +41,11 @@
   @Rule
   public DbTester dbTester = DbTester.create(System2.INSTANCE);
 
-  @Before
-  public void disableIfNotH2() {
```

Comment:
This fix is not enough for supporting all dbs. DbTester#selectFirst() must be fixed for supporting case-sensitive column names. This is part of another branch (PR pushed soon). Meanwhile assumption on H2 should be kept.